src/Handler/Workspace/Drop/DropWorkspaceHandler.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Workspace\Drop;
+
+use Google\Protobuf\Internal\Message;
+use Keboola\StorageDriver\Command\Workspace\DropWorkspaceCommand;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Teradata\TeradataSessionManager;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+use Throwable;
+
+final class DropWorkspaceHandler implements DriverCommandHandlerInterface
+{
+    private TeradataSessionManager $manager;
+
+    public function __construct(TeradataSessionManager $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     * @param DropWorkspaceCommand $command
+     */
+    public function __invoke(
+        Message $credentials, // project credentials
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof DropWorkspaceCommand);
+
+        // validate
+        assert(!empty($command->getWorkspaceUserName()), 'DropWorkspaceCommand.workspaceUserName is required');
+        assert(!empty($command->getWorkspaceRoleName()), 'DropWorkspaceCommand.workspaceRoleName is required');
+        assert(!empty($command->getWorkspaceObjectName()), 'DropWorkspaceCommand.workspaceObjectName is required');
+
+        $db = $this->manager->createSession($credentials);
+
+        try {
+            $db->executeStatement(sprintf(
+                'DROP ROLE %s;',
+                TeradataQuote::quoteSingleIdentifier($command->getWorkspaceRoleName())
+            ));
+
+            $db->executeStatement(sprintf(
+                'DROP USER %s;',
+                TeradataQuote::quoteSingleIdentifier($command->getWorkspaceUserName())
+            ));
+
+            $db->executeStatement(sprintf(
+                'DROP DATABASE %s',
+                TeradataQuote::quoteSingleIdentifier($command->getWorkspaceObjectName()),
+            ));
+        } catch (Throwable $e) {
+            if (!$command->getIgnoreErrors()) {
+                $db->close();
+                throw $e;
+            }
+        }
+
+        $db->close();
```

Comment:
ten db close nemusíš volat nikde pokud používáš session manager

Reply:
pridaval jsem to tam proto, ze kdyz to vyhodilo exception v prubehu, tak to neuzavrelo spojeni - jeste overim...

Reply:
done